src/components/Renderer/index.tsx

```diff
@@ -25,7 +25,7 @@ export const getValue = (
 	schema?: JSONSchema,
 	uiSchema?: UiSchema,
 ) => {
-	const calculatedValue = uiSchema?.['ui:value'];
+	const calculatedValue = uiSchema?.['ui:value'] ?? value;
```

Comment:
The previous refactor on this didn't actually change the behavior at all.
In my attempts, the tests passed even w/o this.

Reply:
Yeah this is wrong as it treats null and undefined the same @JSReds, you can see below we specifically handle null vs undefined separately and do the defaulting to value as need be